FlightStreamDeck.Core/Structs.cs

```diff
@@ -819,6 +819,7 @@ public enum TOGGLE_EVENT
         AUTO_THROTTLE_TO_GA,
         HEADING_BUG_INC,
         HEADING_BUG_DEC,
+        HEADING_BUG_SET,
```

Comment:
Do you mind removing this as I happen to add the same one? Maybe rebase on latest master branch.